fix(dataset): fix augmented dataset

davide-lagamba opened the pull request:
- change dataset separator to ";"
- rename DataAugmentation.ipynb to data_augmentation.ipynb
- modify not readable characters in augmented dataset

Closes #9

davide-lagamba committed:
fix(dataset): fix augmented dataset

- change dataset separator to ";"
- rename DataAugmentation.ipynb to data_augmentation.ipynb
- modify not readable characters in augmented dataset

Closes #9

alfcan reviewed (changes requested):
change intents from 0, 1, 2, 3 to get_smells, get_smells_date, report, info

davide-lagamba committed:
fix(dataset): fix augmented dataset intents

change intents from numeric values to "get_smells, get_smells_date, report, info"
